trade-finance-manager-api/src/graphql/helpers/amendment.helpers.js

```diff
@@ -0,0 +1,189 @@
+const { format, fromUnixTime } = require('date-fns');
+const { formattedNumber } = require('../../utils/number');
+const { decimalsCount, roundNumber } = require('../../v1/helpers/number');
+const { CURRENCY } = require('../../constants/currency');
+const api = require('../../v1/api');
+const { DEALS } = require('../../constants');
+
+// returns the formatted amendment value and currency (without conversion)
+const amendmentChangeValueExportCurrency = (amendment) => {
+  const { currency, value: amendmentValue } = amendment;
+
+  if (currency && amendmentValue) {
+    return `${currency} ${formattedNumber(amendmentValue)}`;
+  }
+
+  return null;
+};
+
+// calculates new facility value in GBP
+const calculateNewFacilityValue = (exchangeRate, amendment) => {
+  const { currency, value: amendmentValue } = amendment;
+  let newValue;
+
+  if (currency && amendmentValue) {
+    // if already in GBP, just take the value
+    if (currency === CURRENCY.GBP) {
+      newValue = amendmentValue;
+    } else {
+      // if no exchange rate return null
+      if (!exchangeRate) {
+        return null;
+      }
+      const valueInGBP = amendmentValue * exchangeRate;
+      const totalDecimals = decimalsCount(valueInGBP);
+      // rounds value to 2dp
+      if (totalDecimals > 2) {
+        newValue = roundNumber(valueInGBP, 2);
+      } else {
+        newValue = valueInGBP;
+      }
+    }
+
+    return newValue;
+  }
+
+  return null;
+};
+
+// calculates new ukef exposure from amendment value
+const calculateUkefExposure = (facilityValueInGBP, coverPercentage) => {
+  let ukefExposure;
+
+  if (facilityValueInGBP && coverPercentage) {
+    // parse float as can have 2dp in facility value
+    const calculation = parseFloat(facilityValueInGBP, 10) * (coverPercentage / 100);
+    const totalDecimals = decimalsCount(calculation);
+
+    if (totalDecimals > 2) {
```

Comment:
Ternary conditional operator can be used here.

Reply:
simplified

Reply:
I can't see the changes here

Reply:
@Zainzzkk the changes that you pushed were for line 24. you didn't push the changes for this line.

Reply:
done